app/models/location.rb

```diff
@@ -40,7 +40,7 @@ def num_checkins
 
   def completed?
     num_teams = Team.playing.count
-    num_checkins == num_teams
+    num_checkins >= num_teams
```

Comment:
I don't disagree with the logic, but I'm curious how you would get into a state where the number of checkins exceeded the number of teams? If that were the case, it's possible for the location to be considered completed before it actually is.